platform/remarkable/README_remarkable.txt

```diff
@@ -13,12 +13,35 @@ with a USB cable.
    ssh root@<your-remarkable>
 
    unzip <koreader-package>.zip
+
+# Launching using the included launcher
+
+koreader includes a very simple launcher called `button-listen` which starts
+koreader when the middle button is held down for 3 or more seconds in the
+xochitl UI.
```

Comment:
Do people know what that is? A subtle hint like "the default xochitl UI" might be preferable.

Reply:
Good point, I've mentioned xochitl the first time and just used Default UI from then on.